bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -35,72 +24,68 @@ impl WasmBezier {
 	/// Expect js_points to be a list of 3 pairs
 	pub fn new_quad(js_points: &JsValue) -> WasmBezier {
 		let points: [DVec2; 3] = js_points.into_serde().unwrap();
-		WasmBezier {
-			internal: Bezier::from_quadratic_dvec2(points[0], points[1], points[2]),
-		}
+		WasmBezier(Bezier::from_quadratic_dvec2(points[0], points[1], points[2]))
 	}
 
 	/// Expect js_points to be a list of 4 pairs
 	pub fn new_cubic(js_points: &JsValue) -> WasmBezier {
 		let points: [DVec2; 4] = js_points.into_serde().unwrap();
-		WasmBezier {
-			internal: Bezier::from_cubic_dvec2(points[0], points[1], points[2], points[3]),
-		}
+		WasmBezier(Bezier::from_cubic_dvec2(points[0], points[1], points[2], points[3]))
 	}
 
 	pub fn set_start(&mut self, x: f64, y: f64) {
-		self.internal.set_start(DVec2::from((x, y)));
+		self.0.set_start(DVec2::from((x, y)));
```

Comment:
Oh yeah, I'm wondering if we want to switch to using `DVec2::new(x, y)` instead of having to create a tuple i.e. this'll get rid of some brackets.

Reply:
Fixed all occurrences of this issue in wasm/src/lib.rs and lib/src/lib.r